src/Luxifer/DQL/String/IfElse.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Luxifer\DQL\String;
+
+use Doctrine\ORM\Query\AST\Functions\FunctionNode;
+use Doctrine\ORM\Query\Lexer;
+use Doctrine\ORM\Query\Parser;
+use Doctrine\ORM\Query\SqlWalker;
+
+/**
+ * ConcatWsFunction ::= "IF" "(" ConditionalExpression "," ArithmeticExpression "," ArithmeticExpression ")"
+ */
+class IfElse extends FunctionNode
```

Comment:
The SQL keyword is `IF` but in reality it's an if/else control flow because there's an `if true` statement and an `if false`